src/platform/silabs/SiWx917/CHIPCryptoPALTinyCrypt.cpp

```diff
@@ -1523,7 +1497,7 @@ CHIP_ERROR ExtractPubkeyFromX509Cert(const ByteSpan & certificate, Crypto::P256P
     VerifyOrExit(mbedtls_pk_get_type(&(mbed_cert.CHIP_CRYPTO_PAL_PRIVATE_X509(pk))) == MBEDTLS_PK_ECKEY,
                  error = CHIP_ERROR_INVALID_ARGUMENT);
 
-    keypair                    = mbedtls_pk_uecc(mbed_cert.CHIP_CRYPTO_PAL_PRIVATE_X509(pk));
+    keypair                    = (mbedtls_uecc_keypair *) (mbed_cert.CHIP_CRYPTO_PAL_PRIVATE_X509(pk)).pk_ctx;
```

Comment:
I was not able to understand the context here to use `mbedtls_x509_crt_verify`, since this API looks like it will be invoked to verify the format of the certificate. Hence replaced with `mbedtls_pk_ec` instead of using private members.